install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -54,10 +54,29 @@ the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
 in {product-title} 3.9. The following examples show configuration files with the
 changed values.
 
-.Considerations when using MySQL and PostgreSQL with Azure file
-* The owner UID of the Azure File mounted directory is different from the UID of a container.
-* MySQL containers change the file owner permission in the mounted directory. Because of the mismatch between the owner UID and container process UID, this operation fails. Therefore to run MySQL with Azure File:
-** Specify the Azure File mounted directory UID in the `runAsUser` variable in the PV configuration file.
+.Considerations when using Azure File
+
+The following file system features are not supported by Azure File:
+
+* Symlinks
+* Hard links
+* Extended attributes
+* Sparse files
+* Named pipes
+
+Additionally, the owner UID of the Azure File mounted directory is different
+from the process UID of the container.
+
+You may experience instability in your environment if you use any container
+images that utilize unsupported file system features. Containers for PostgreSQL
+and MySQL are known to have issues when used with Azure File.
+
+For MySQL containers, the container changes the file owner permission in the
```

Comment:
I'd rewrite this section to start with ". If you use MySQL containers, you must modify the PV configuration file..."
I might use two substeps for the different modifications. I'd also clarify what the two sample files (.Storage class configuration file example and .PV configuration file example) are for.